README.md

```diff
@@ -1,10 +1,31 @@
-# digital-fuesim-manv
+# Digitale FüSim MANV
 
-The official code for BP2021HG1
+This is the codebase for a digital implementation of the "FüSim MANV" (Führungssimulation Massenanfall von Verletzen), a German simulation system for training emergency medical services leadership personnel on how to manage [Mass Casualty Incidents](https://en.wikipedia.org/wiki/Mass-casualty_incident).
 
-You can find the (internal) documentation for this project [here](https://github.com/hpi-sam/BP2021HG1).
+**You can try it out at [https://fuesim-manv.de/](https://fuesim-manv.de/)**.
 
-The (internal) project-board is [here](https://github.com/orgs/hpi-sam/projects/4).
+![image](https://user-images.githubusercontent.com/18506183/172071147-24b9aabe-51ee-4105-a5a4-6cbf8063eece.png)
```

Comment:
Is it common practice to host images this way on some GitHub server? or should the image be in the repo?